pkg/util/purge/purge.go

```diff
@@ -0,0 +1,56 @@
+package purge
```

Comment:
I'm not sure that I want to see this package in `pkg/util`: we are likely not going to use it anywhere other than cleanup script. Maybe we should keep it somewhere under `hack/clean`?

Reply:
This way we can split it better and use `pkg` format. Doing this in hacks makes it not nice to read and a little bit of a mess.